### ISSUE TITLE
Error with "intersects" method when using Feature Collections.

### ISSUE THREAD

**Issue by rgwozdz** [terraformer-js/terraformer#83]
When attempting an [`intersects` with multipolygon and polygon arguments](https://runkit.com/rgwozdz/61e094154b4dbe0008a6abfc), I get a fatal error:

```js
const { intersects }= require("@terraformer/spatial")

const multipolygon = {"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"MultiPolygon","coordinates":[[[[-101.0,37.0],[-99.0,37.0],[-99.0,38.0],[-101.0,38.0],[-101.0,37.0]]],[[[-97.0,35.0],[-95.0,35.0],[-95.0,36.0],[-97.0,36.0],[-97.0,35.0]]]]}},{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-104,33],[-100,33],[-100,34],[-104,34],[-104,33]]]}}]}

const geometryFilter = {"type":"FeatureCollection","features":[{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-98.5,34.6],[-94,34.6],[-94,37],[-98.5,37],[-98.5,34.6]]]}}]}

intersects(multipolygon, geometryFilter)
```

Results:

![Screen Shot 2022-07-05 at 1 14 23 PM](https://user-images.githubusercontent.com/4369192/177408695-a0a1091c-e84c-4565-963d-698e4ac27d90.png)

**Comment by rgwozdz** [comment]
@jgravois - ok, this isn't a big deal to me; I actually happened upon the error as I was digging into some other behavior with `intersects`.  I'm happy to leave as is, but maybe we should update the [doc](https://github.com/terraformer-js/terraformer/blob/main/packages/spatial/README.md#terraformerintersectsgeojson-geojson--boolean) as it appears to state that FeatureCollection is an allowable input.
If you think that update is a good idea, let me know, and I can PR.

**Comment by jgravois** [comment]
> this isn't a big deal to me

understood. i definitely do appreciate you making notes and leaving breadcrumbs as you work.

PRs with tweaks to the doc in the name of clarification are always welcome. if you or a colleague end up poking around in `intersects()` code someday and want to codify the expectation with regard to FeatureCollections clear there too, all the better. 🙏

fwiw, something simple to document the behavior in https://github.com/terraformer-js/terraformer/issues/71 is warranted too.

**Comment by rgwozdz** [comment]
Cool will do; updated the name of the issue here.  I'm actually seeing something possibly(?) unexpected with `intersects` when I pass in simple multipolygon geometries.  I'll put it in a a fresh issue though.

**Comment by jgravois** [comment]
my plan is to publish a release as soon as this small doc issue and #68 are addressed.

**Comment by jgravois** [comment]
resolved by #89 🌹
